src/ecrc-frontend/src/components/page/consent/Consent.js

```diff
@@ -228,200 +222,41 @@ export default function Consent({
       eivPassDetailsResults: "eivPassDetailsResults"
     };
 
+    const singleCall = {
+      requestGuid: uuid,
+      approvedPage: `${process.env.REACT_APP_FRONTEND_BASE_URL}/criminalrecordcheck/success`,
+      declinedPage: `${process.env.REACT_APP_FRONTEND_BASE_URL}/criminalrecordcheck/success`,
+      errorPage: `${process.env.REACT_APP_FRONTEND_BASE_URL}/criminalrecordcheck/success`,
```

Comment:
question: since all these are same, can we pass just one?

Reply:
We should be able to make this change if coordinated with API.